README.md

```diff
@@ -33,34 +53,11 @@ This works for arrays as well, for example the list of enabled render states.  H
-* Previewing embedded JPG and PNG images works.
-* Previewing buffer data is not (yet?) supported.
-
-Note that previewed shaders are displayed with a `.glsl` file extension.  It is recommended that you install a 3rd-party syntax highlighter with support for `*.glsl`, for example [Shader Language Support for VSCode by slevesque](https://marketplace.visualstudio.com/items?itemName=slevesque.shader), to enable syntax highlighting in shader previews.
-
-Note that many dataURIs are quite long and can look bad in the editor.  Hover your mouse over the line numbers in the left margin to reveal little "`-`" icons that can fold closed blocks of JSON.  Fold closed the innermost block containing the `uri` field.  You can still select the name of the block that got folded and press <kbd>ALT</kbd> + <kbd>D</kbd> to preview the contents of that uri without unfolding it.
+## Compatibiliy and known size limitations
 
-### &bull;  Import and export embedded dataURIs
+Currently, this extension is compatible only with `*.gltf` files, not the binary form of `*.glb`.  This may change in the future.  [Contributions](CONTRIBUTING.md) welcome!
 
-In the list of commands (<kbd>CTRL</kbd> + <kbd>SHIFT</kbd> + <kbd>P</kbd>), there are two commands named `Import file as Data URI` and `Export a Data URI to a file`.  To use these, place the document cursor on a block that contains a `"uri"` field.  If the value of this field is a valid filename, `Import` will load that file, encode it to base64, and replace the filename with the dataURI in your document.  `Export` is the reverse of this process, but first it will ask you for a filename to save to.  It will save the file in the same folder as the glTF file, so it does not need a path, just a name.  It will try to select an appropriate file extension based on the MIME type of the dataURI.  It will also warn you if you are about to overwrite an existing file.  If the save is successful, the dataURI will be replaced by the name of the newly created file.
+Also, there is a [known issue](https://github.com/Microsoft/vscode/issues/31078) where VSCode extensions cannot manipulate documents that have grown beyond 5 MB in size.  This limitation applies only to the main JSON-based glTF document, not to external assets such as the `bin` file, texture image files, etc.  The sample model repo contains some models called `glTF-Embedded`, where all of the assets have been URI-encoded and placed inside the JSON document.  For some of the larger models, this does make the document larger than the size limit, preventing this extension's preview windows from opening.  We hope this limit will be increased or made user-configurable in the future.  In the meantime, the workaround is to use the non-embedded versions of those models.  The non-embedded versions are all well below the size limit, because the bulk of their data is external to the JSON.
```

Comment:
I would also add something like: "If a non-embedded version of your model is not available to you, can you always use [the export feature](#convert-files-to-and-from-data-uris) to export the larger embedded content in order to get your json file under the 5mb limit."